jbmc/regression/jbmc-strings/string-input-value/test1.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+Test.class
+--verbosity 10 --string-input-value fo --string-input-value barr --function Test.checkStringInputValue --string-input-value "" --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
```

Comment:
:pick: Could we also have a test.desc where "" (and maybe others) are not given as string-input-value to see the difference?